src/test/java/com/cloudbees/jenkins/support/impl/RunningBuildsTest.java

```diff
@@ -0,0 +1,70 @@
+package com.cloudbees.jenkins.support.impl;
+
+import com.cloudbees.jenkins.support.SupportTestUtils;
+import hudson.model.FreeStyleBuild;
+import hudson.model.FreeStyleProject;
+import hudson.model.queue.QueueTaskFuture;
+import junit.framework.AssertionFailedError;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.jenkinsci.plugins.workflow.test.steps.SemaphoreStep;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.MockFolder;
+
+import java.util.Optional;
+
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.assertThat;
+
+public class RunningBuildsTest {
+
+    private static final String JOB_NAME = "job-name";
+    private static final String FOLDER_NAME = "folder-name";
+    private static final String EXPECTED_OUTPUT_FORMAT = "%s #%d";
+    private static final String EXPECTED_FOLDER_OUTPUT_FORMAT = "%s/" + EXPECTED_OUTPUT_FORMAT;
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @Test
+    public void addContents() throws Exception {
+        FreeStyleProject p = j.createFreeStyleProject(JOB_NAME);
+        QueueTaskFuture<FreeStyleBuild> freeStyleBuildQueueTaskFuture = p.scheduleBuild2(0);
+        FreeStyleBuild build = freeStyleBuildQueueTaskFuture.waitForStart();
+
+        String output = SupportTestUtils.invokeComponentToString(new RunningBuilds());
+
+        assertThat(output, containsString(String.format(EXPECTED_OUTPUT_FORMAT, p.getName(), build.getNumber())));
```

Comment:
So is this eventually filtered or not?

Reply:
I don't really understand your question. Sorry.

Reply:
The test is checking the actual job name is displayed in the output string, when that should be something anonymized. Is it that the info is not anonymized or that the utility test function is not taking that into account?

Reply:
I'll add a test for anonymization.

Reply:
https://github.com/jenkinsci/support-core-plugin/pull/189/commits/6d986a5207a3633930088c71c9f90cacbf090018